pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1051,22 +1051,14 @@ self: super: {
   # `removeTrailingWhitespace` is buggy in earlier versions.
   # This will probably be able to be removed when we update to LTS-15.
   dhall_1_29_0 =
-    dontCheck (super.dhall_1_29_0.override {
+    dontCheck (super.dhall_1_29_0.overrideScope (self: super: {
       prettyprinter = self.prettyprinter_1_6_0;
-      prettyprinter-ansi-terminal =
-        self.prettyprinter-ansi-terminal.override {
-          prettyprinter = self.prettyprinter_1_6_0;
-        };
-    });
+    }));
```

Comment:
Thanks for cleaning this up.

I didn't realize that `overrideScope` was the tool to use here, but after seeing this example, I'll try to use it when appropriate!